src/assets/styles/_host.scss

```diff
@@ -6,5 +6,5 @@
   }
   background-color: var(--calcite-ui-foreground-1);
   color: var(--calcite-ui-text-2);
-  @include font-size(-2);
+  font-size: var(--calcite-font-size--1);
```

Comment:
if apply works here, it'd be `@apply box-border bg-foreground-1 text-color-2 text--1`